src/metpy/calc/thermo.py

```diff
@@ -946,13 +951,6 @@ def _parcel_profile_helper(pressure, temperature, dewpoint,
     press_lower = concatenate((pressure[pressure >= press_lcl], press_lcl))
     temp_lower = dry_lapse(press_lower, temperature)
```

Comment:
Here's the problem. The temperature being passed in here is the virtual temperature; it should still be the regular temperature. After getting the temp_lower, *then* you compute the virtual temperature of the parcel path.